libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/MediaUtils.java

```diff
@@ -170,7 +170,8 @@ public static boolean isInMediaStore(Uri mediaUri) {
     }
 
     public static @Nullable String getFilenameFromURI(Context context, Uri uri) {
-        Cursor cursor = context.getContentResolver().query(uri, null, null, null, null);
+        Cursor cursor = context.getContentResolver().query(uri, new String[]{OpenableColumns.DISPLAY_NAME},
+                null, null, null);
```

Comment:
This the only line that matters for the solution @khaykov

Reply:
This crash is occurring because, in API 29, [they depreciated the latitude column](https://developer.android.com/reference/android/provider/MediaStore.Images.ImageColumns#LATITUDE). 
So the query is currently calling for all columns including the depreciated column even though it isn't used.